lib/screens/detail_page.dart

```diff
@@ -1,80 +1,200 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+import 'package:provider/provider.dart';
+import 'package:timeago/timeago.dart' as timeago;
 import 'package:todo_app/screens/edit_note_page.dart';
 import 'package:todo_app/src/constants.dart';
+import 'package:todo_app/src/models/note.dart';
+import 'package:todo_app/src/provider/provider_data.dart';
 import 'package:todo_app/src/services.dart';
+import 'package:todo_app/widgets/checkbox_widget.dart';
 
+///
 class DetailPage extends StatefulWidget {
+  ///
   final Note? note;
 
-  const DetailPage({this.note});
+  ///
+  final int noteId;
+
+  ///
+  const DetailPage({this.note, required this.noteId});
   @override
   _DetailPageState createState() => _DetailPageState();
 }
 
 class _DetailPageState extends State<DetailPage> {
-  Color _colText = textColorK;
   List<Todo> taskList = [];
   @override
   void initState() {
-    _colText = _updateColor();
+    // _noteInfo();
     super.initState();
   }
 
-  // Future<List<Todo>> _getTask() async {
-  //   final _dataTasks =
-  //       Services.of(context).notesService.getTaskById(widget.note?.id ?? 0);
+  @override
+  Widget build(BuildContext context) {
+    return const DetailPageFuture();
+  }
 
-  //   return _dataTasks;
-  // }
+  @override
+  void dispose() {
+    super.dispose();
+  }
+}
 
-  Color _updateColor() {
-    final Color dataColor = ThemeData.estimateBrightnessForColor(
-              colorPallete[widget.note?.colorNote ?? 0],
-            ) ==
-            Brightness.light
-        ? const Color.fromARGB(255, 97, 85, 85)
-        : Colors.white;
+///
+class DetailPageFuture extends StatefulWidget {
+  ///
+  const DetailPageFuture({
+    Key? key,
+    //this.noteInfo,
+  }) : super(key: key);
 
-    return dataColor;
-  }
+  @override
+  State<DetailPageFuture> createState() => _DetailPageFutureState();
+}
 
+class _DetailPageFutureState extends State<DetailPageFuture> {
+  ///
   void _showSnackBar(String text) {
     ScaffoldMessenger.of(context).showSnackBar(SnackBar(content: Text(text)));
   }
 
+  ///
+  void _checkInputText(String taskStr) {
+    if (taskStr.isEmpty) {
+      return _showSnackBar('Please, enter some text');
+    }
+    if (taskStr.isNotEmpty) {
```

Comment:
The second check is redundant -- the part outside the curly brackets will execute only if taskStr.isNotEmpty anyway.
```suggestion
    if (taskStr.isEmpty) {
      return _showSnackBar('Please, enter some text');
    }
```